index.html

```diff
@@ -160,13 +200,139 @@
             </header>
             <div class="top_banner_content">
                 <h1 id="future">We Take Care of the Future</h1>
-                <h4 id="weknow">We know how large objects will act, but things on a small scale just do not act that way</h4>
+                <h4 id="weknow">We know how large objects will act, but things on a small scale just do not act that way
+                </h4>
                 <div class="top_banner_actions">
                     <button class="bttn bttn--primary">Get Quote Now</button>
                     <button class="bttn bttn--secondary">Learn More</button>
                 </div>
             </div>
         </div>
     </section>
+    <style>
```

Comment:
Having style right inside the body is not the best idea
Move it into `<style>` section into head or in a separate file